systems/subscriber/registry/pkg/config.go

```diff
@@ -41,6 +48,18 @@ func NewConfig(name string) *Config {
 		Service: uconf.LoadServiceHostConfig(name),
 		MsgClient: &uconf.MsgClient{
 			Timeout: 5 * time.Second,
+			ListenerRoutes: []string{
+				"event.cloud.local.{{ .Org}}.subscriber.simmanager.sims.deletion_completed",
```

Comment:
We shoud follow routing keys convention: no underscore  "sims.delete" seems fine.

Reply:
Addressed.